druid/src/widget/invalidation.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 The xi-editor Authors.
```

Comment:
This feels like an example? I'm not sure it belongs as a core widget.

Reply:
This is in support of `WidgetExt::debug_invalidation`. I originally tried it using `EnvScope`, but I don't think you want the invalidation regions to be drawn recursively so I ended up breaking it into another widget.